README.md

```diff
@@ -2,5 +2,62 @@
 
 Terraform module that will create an S3 bucket in AWS with relevant user account that will have access to bucket.
 
+The bucket created will use $team_name-$bucket_identifier as the bucket name to ensure the bucket created is globally unique and avoid name clashes with buckets.
```

Comment:
Please use backticks for things like this, it makes is much easier to read, eg: `${team_name}-${bucket_identifier}`.